src/Data/Cache.hs

```diff
@@ -71,15 +71,15 @@ withNullCache body = body nullCache
 
 
 -- | A very inefficient example implementation.
--- 
+--
 -- >>> withAssocCache testC
 -- one
 -- two
 -- testing
 -- [3,3,3,3,7,7]
 assocCache :: (Monad m, Eq k) => Cache (StateT [(k,a)] m) k a
 assocCache = Cache
```

Comment:
Elsewhere, you replaced `fmap` by `(<$>)`. Is there any particular reason why you prefer `fmap` to `(<$>)` here but not over there?